python/dbos-app-starter/html/app.html

```diff
@@ -97,6 +98,10 @@ <h1 class="text-xl font-semibold mb-4">Welcome to DBOS!</h1>
                 }
                 const step = await response.text();
                 document.getElementById('status').innerHTML = `Your background task has completed <b>${step} of 10</b> steps`;
+                if (step == 10) {
+                    currentId = null
+                    urlParams.set('id', currentId);
```

Comment:
I don’t think we need to set urlParams here

Reply:
I tried that first. The problem is that if we don't set it, it is read as part of the function and currentId is not reset

Reply:
... no I think you may be right.... testing...

Reply:
Yep... you're right. I had a brainfart. Fixed.